### PR TITLE
docs: add instructions to get transaction id

### PR DESCRIPTION
**Pull request by mamoonraja**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -401,6 +401,29 @@ private void OnMessage(DetailedResponse<MessageResponse> response, IBMError erro
 }
 ```
 
+## Transaction IDs
+Every SDK call will return a response which will contain a transaction ID, accessible via the `x-global-transaction-id` header. This transaction ID is useful for troubleshooting and accessing relevant logs from your service instance.
```
Comment (mediumTaj) [inline README.md]: @SirSpidey chimed in [here](https://github.com/watson-developer-cloud/dotnet-standard-sdk/pull/413) and offered updated messaging that better conforms to IBM's documentation style guide.

>Every SDK call returns a response with a transaction ID in the x-global-transaction-id header. This transaction ID is useful for troubleshooting and accessing relevant logs from your service instance.

File: README.md [hunk 2]
```diff
@@ -401,6 +401,29 @@ private void OnMessage(DetailedResponse<MessageResponse> response, IBMError erro
...
+            Log.Debug("AssistantServiceV1", "Global transaction id: {0}", globalTransactionId);
+            listWorkspacesResponse = response.Result;
+        }
+    );
+
+    while (listWorkspacesResponse == null)
+        yield return null;
+}
+```
+
```
Comment (mediumTaj) [inline README.md]: Do you get the transaction id if you have an error as well? ie if you call GetWorkspace with a bad workspaceId?

Comment (mamoonraja) [reply README.md]: updated.

**Pull request merged**